source/process/documentation-UItext-guidelines.rst

```diff
@@ -0,0 +1,185 @@
+User Interface Text Guidelines
+========================
+This is the Mattermost style guide for text used in the Mattermost product. Use these guidelines to ensure the the text on the Mattermost user interface (UI) is consistent and clear.
+
+.. note::
+
+These guidelines are not intended to slow down or otherwise impede contributions, which are always welcome. No contribution will be rejected due to non-conforming style, although it might be edited.
+
+....
+
+.. contents:: 
+   :depth: 2
+
+....
+
+UI elements
+------------
+Use this table when writing the text for UI elements such as windows and dialog boxes.
+
+.. list-table::
+   :widths: 10 30 10 15 35
+   :header-rows: 1
+
+   * - Element
+     - Image 
+     - Capitalization
+     - Phrasing
+     - Examples     
+   * - Menu
+     - |menu.png|
+     - Headline
+     - - Noun, noun phrase, or verb
+       - No punctuation
+       - Not more than three words
+     - - *Members*
+       - *Account Preferences*
+       - *Log Out*
+   * - Tooltip
+     - |tooltip.png|
+     - Sentence
+     - - Sentence fragment or sentence
+       - No punctuation
+       - Include articles (*a*, *an*, *the*)
+     - - *Start a Zoom meeting*
+       - *Flag for following up*
+       - *Remove from this list*
+   * - In-field text
+     - |field.png|  
+     - Sentence
+     - - Sentence fragment, sentence, or word
+       - No punctuation
+       - Include articles (*a*, *an*, *the*)
+     - - *Add a comment*
+       - *Search*
+   * - Action button
+     - |action.png|
+     - Headline
+     - - Verb or verb phrase
+       - No articles (*a*, *an*, *the*)
+       - Exceptions: *OK*, *Yes*, *No*
+     - - *Add Comment*
+       - *Edit*
+   * - Label before a UI element
```

Comment:
I don't think we need to establish that hierarchy. For example, the Account Settings option is a modal window with its own left-navigation (which will follow the Menu guidelines), and each of the options (Notifications, e.g.) is a menu item. The options within these menu items (e.g. Desktop notifications, Email notification, etc.) are also menu items according to me (because they have an edit option). What do you think?
Or, did I understand the comment wrong?

Reply:
Ignore my prev. reply, please. Eric added a screenshot to illustrate this, and i commented there...